jekyll/_cci2/vm-service.adoc

```diff
@@ -66,20 +66,15 @@ image::vm-service-custom.png[Custom VM Service Images]
 * Packer (https://packer.io/intro/getting-started/install.html)
 * AWS Access Key ID and Secret Access Key
 
-==== Creating a Custom AMI
+==== Creating a Custom Linux AMI
 
 1. Clone our image builder repo: https://github.com/circleci/image-builder/tree/picard-vm-image
```

Comment:
I know you didn't make a change to this line, but could we make this an inline link?

Reply:
On the site it is a link:
<img width="568" alt="Screenshot 2020-09-16 at 21 19 28" src="https://user-images.githubusercontent.com/24589403/93388244-53861100-f862-11ea-8ac6-899c1c69abc0.png">

or do you mean [image builder repo](URL) style?

Reply:
@teesloane ^

Reply:
Thanks for the review BTW!